docs/security-champion/1-new_security_champion.md

```diff
@@ -0,0 +1,34 @@
+# I've joined, now what?
+
+## Welcome young padawan
```

Comment:
Are we really going wit Star Wars references? Didn't it stop being cool after Disney?